web-client/src/presenter/computeds/fileUploadStatusHelper.test.js

```diff
@@ -0,0 +1,82 @@
+import { runCompute } from 'cerebral/test';
+
+import { fileUploadStatusHelper } from './fileUploadStatusHelper';
+
+describe('fileUploadStatusHelper', () => {
+  it('returns `Preparing Upload` for infinity time remaining', async () => {
+    const result = await runCompute(fileUploadStatusHelper, {
+      state: {
+        isUploading: true,
+        timeRemaining: Number.POSITIVE_INFINITY,
+      },
+    });
+
+    expect(result.statusMessage).toEqual('Preparing Upload');
+  });
+
+  it('returns `Less Than 1 Minute Left` for anything less than 60 seconds', async () => {
+    const result = await runCompute(fileUploadStatusHelper, {
+      state: {
+        isUploading: true,
+        timeRemaining: 40,
+      },
+    });
+
+    expect(result.statusMessage).toEqual('Less Than 1 Minute Left');
+  });
+
+  it('returns `1 Minutes Left` for time remaining being something between 60 and 3600 seconds', async () => {
```

Comment:
Shouldn't this read `between 1 and 60 seconds`? (I imagine you were thinking of microseconds?)